dev-vcs/fossil/fossil-2.16.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A tool for distributed software configuration management"
+DESCRIPTION="Fossil is a simple, high-reliability, distributed version \
+control system like Git and Mercurial, but Fossil also supports distributed \
+bug tracking, distributed wiki, distributed forums, and a distributed blog \
+mechanism all in a single integrated package.
+
+Additionally, Fossil also has a built-in and easy-to-use web interface that \
+simplifies project tracking and promotes situational awareness."
+HOMEPAGE="https://www.fossil-scm.org/"
+COPYRIGHT="2007-2021 D. Richard Hipp"
+LICENSE="BSD (2-clause)"
+REVISION="2"
+SOURCE_URI="https://fossil-scm.org/home/tarball/7aedd5675883d4412cf20917d340b6985e3ecb842e88a39f135df034b2d5f4d3/fossil-src-$portVersion.tar.gz"
```

Comment:
Maybe we could use something like they used in Arch? https://github.com/archlinux/svntogit-community/blob/packages/fossil/trunk/PKGBUILD (not checked)
eg:
```diff
- SOURCE_URI="https://fossil-scm.org/home/tarball/7aedd5675883d4412cf20917d340b6985e3ecb842e88a39f135df034b2d5f4d3/fossil-src-$portVersion.tar.gz"
+ tag="7aedd5675883d4412cf20917d340b6985e3ecb842e88a39f135df034b2d5f4d3"
+ SOURCE_URI="https://fossil-scm.org/home/tarball/$tag/fossil-src-$portVersion.tar.gz"
```

Reply:
Thanks for the review!

Updated the commit to use $tag.  The variable "tag" seems out-of-place since other variables are upper-case. However, "tag" is non-standard and this is probably bike-shedding...